ludwig/features/category_feature.py

```diff
@@ -126,8 +125,8 @@ def get_feature_meta(column, preprocessing_parameters, backend):
         )
         vocab_size = len(str2idx)
         if vocab_size <= 1:
-            raise InputDataError(
-                column.name, CATEGORY, f"At least 2 distinct values are required, column only contains {str(idx2str)}"
+            logger.warning(
+                f"Feature {column.name} of type {CATEGORY} only contains {vocab_size} distinct values: {str(idx2str)}."
```

Comment:
This warning by itself doesn't say much. I'd probably expand and add in that this feature will not be useful for the machine learning model and that they should consider dropping it